### PR TITLE
Add discription to the function

### PR DESCRIPTION
**Pull request by fatma2585**

### CODE REVIEW

File: participants/Shaban/pr_tutorial/simple_functions.py [hunk 1]
```diff
@@ -5,8 +5,14 @@ def fibonacci(max):
     return values
 
 
-def factorial(value):
+def factorial(value):
```
Comment (marixko) [inline participants/Shaban/pr_tutorial/simple_functions.py:8]: looks great

Comment (marixko) [inline participants/Shaban/pr_tutorial/simple_functions.py:8]: looks good

**Review by marixko — APPROVED** [review]

File: participants/Shaban/pr_tutorial/simple_functions.py [hunk 2]
```diff
@@ -5,8 +5,14 @@ def fibonacci(max):
...
+    '''
+    This function calculates the factorial
+
+    value: float input
+    '''  
     if value == 0:
         return 1
     else:
         return value * factorial(value - 1)
+
```
Comment (camipacifici) [inline participants/Shaban/pr_tutorial/simple_functions.py:18]: No need for this additional line.

**Review by camipacifici — APPROVED** [review]
Found just a super minor detail.
Looks great.

**Comment by ceb8** [comment]
Thanks! 😄

**Pull request merged**